packages/frontend-api/src/Model/Resolver/Products/ProductResolver.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Model\Resolver\Products;
+
+use Overblog\GraphQLBundle\Definition\Resolver\AliasedInterface;
+use Overblog\GraphQLBundle\Definition\Resolver\ResolverInterface;
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Model\Product\ProductFacade;
+
+class ProductResolver implements ResolverInterface, AliasedInterface
+{
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Product\ProductFacade
+     */
+    protected $productFacade;
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Domain\Domain
+     */
+    protected $domain;
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Product\ProductFacade $productFacade
+     * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
+     */
+    public function __construct(
+        ProductFacade $productFacade,
+        Domain $domain
+    ) {
+        $this->productFacade = $productFacade;
+        $this->domain = $domain;
+    }
+
+    /**
+     * @param mixed $uuid
```

Comment:
Why is this mixed?

Reply:
Fixed https://github.com/shopsys/shopsys/pull/1471/commits/7cb917ff21f40f43c957bbd6cb31ec2c3e7f8458